portfolio/main/models.py

```diff
@@ -33,6 +33,9 @@ def __str__(self):
 
 class Entry(Page, TimeStampedModel):
 
+    affiliation = models.CharField(
+        help_text='What department is this project for?',
```

Comment:
@zmustapha if I could humbly suggest, I think this would be more clear if it were 'Which department is this project for?'

Reply:
Not now. I'll let this be later.